stdlib/public/Concurrency/PartialAsyncTask.swift

```diff
@@ -909,3 +928,17 @@ public func _unsafeInheritExecutor_withUnsafeThrowingContinuation<T>(
 public func _abiEnableAwaitContinuation() {
   fatalError("never use this function")
 }
+
+@available(StdlibDeploymentTarget 6.2, *)
+@_silgen_name("_swift_createJobForTestingOnly")
+public func _swift_createJobForTestingOnly(
```

Comment:
Kindof wondering if making it @_spi(ConcurrencyTesting) or something similar might be warranted in this case?